op5build/ci_config.yml

```diff
@@ -42,9 +42,11 @@ post:
       ln -s /usr/bin/resolveip /usr/libexec/resolveip
     fi
 
-    # skip systemd tests on EL6
-    if [ -f /usr/bin/systemctl ]; then
-      cucumber --strict --format html --out /mnt/logs/cucumber.html --format pretty
-    else
-      cucumber --strict --format html --out /mnt/logs/cucumber.html --format pretty --tags ~@systemd
-    fi
+    cucumber --strict --format html --out /mnt/logs/cucumber.html --format pretty
+    # Run the test suite again, but with encryption enabled
+    # Some tests has been disabled. These tests pass just fine when running the
+    # test suite with encryption enabled only, but for some reason fail when
+    # running after the above test, without encryption enabled. Further these
+    # tests all have peers/pollers so encryption shouldn't affect them.
```

Comment:
Isn't it between peers and pollers we want test encryption?

Reply:
Right - that is a "typo". The disabled tests do not have any peers/poller.

Updated.